internal/provisioner/objects/deployment/deployment.go

```diff
@@ -105,6 +105,7 @@ func DesiredDeployment(contour *model.Contour, image string) *appsv1.Deployment
 		switch {
 		case port.Name == "http" && port.ContainerPort != objects.EnvoyInsecureContainerPort:
 			args = append(args, fmt.Sprintf("--envoy-service-http-port=%d", port.ContainerPort))
+
```

Comment:
Let's revert the no-op changes to deployment.go and deployment_test.go, for clarity in the commit.